app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -136,29 +136,33 @@ public function panel(Panel $panel): Panel
             ])
             ->navigationGroups([
                 NavigationGroup::make()
-                    ->label('Artificial Intelligence')
-                    ->icon('heroicon-o-sparkles')
+                    ->label('Clients')
+                    ->icon('heroicon-o-users')
                     ->collapsed(),
                 NavigationGroup::make()
-                    ->label('Recruitment CRM')
-                    ->icon('heroicon-o-magnifying-glass')
-                    ->collapsed(),
-                NavigationGroup::make()
-                    ->label('Engagement Features')
-                    ->icon('heroicon-o-signal')
+                    ->label('Service Management')
+                    ->icon('heroicon-o-wrench-screwdriver')
                     ->collapsed(),
                 NavigationGroup::make()
-                    ->label('Premium Features')
-                    ->icon('heroicon-o-rocket-launch')
+                    ->label('Staff Engagement')
+                    ->icon('heroicon-o-users')
                     ->collapsed(),
                 NavigationGroup::make()
-                    ->label('Reporting')
+                    ->label('Report Center')
                     ->icon('heroicon-o-document-chart-bar')
                     ->collapsed(),
                 NavigationGroup::make()
                     ->label('Product Administration')
                     ->icon('heroicon-o-wrench-screwdriver')
                     ->collapsed(),
+                NavigationGroup::make()
+                    ->label('Engagement Features')
+                    ->icon('heroicon-o-signal')
+                    ->collapsed(),
+                NavigationGroup::make()
+                    ->label('Premium Features')
+                    ->icon('heroicon-o-rocket-launch')
+                    ->collapsed(),
```

Comment:
Let's put these two above Product Administration so that Product Administration is always at the bottom.

Reply:
Done